### PR TITLE
test: adding components tests - RadioGroup

### PR DESCRIPTION
**Pull request by Manuela-Sousa**
# Description

This PR includes component tests for storybook component radioGroup. The tests are using storybook test runner validate the renderization of each story from storybook and also implement a play function to validate interactions with the component.
https://storybook.js.org/blog/test-component-interactions-with-storybook/

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Other:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by jessywork — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "test:watch": "jest --watch",
-    "prettier:fix": "prettier --write --ignore-path .gitignore ."
+    "prettier:fix": "prettier --write --ignore-path .gitignore .",
+    "test-storybook": "test-storybook"
```
Comment (joaoprcosta) [inline package.json:33]: To follow the pattern used in other scripts.

```suggestion
    "test:storybook": "test-storybook"
```

**Comment by joaoprcosta** [comment]
Commit message is still in 'feat' and not 'test'.

File: package.json (hunk 1 above)
Comment (Manuela-Sousa) [reply package.json]: Alterado

**Comment by Manuela-Sousa** [comment]
changed message as well

**Comment by Manuela-Sousa** [comment]
changed commit message as well to test

**Review by joaoprcosta — APPROVED** [review]

**Pull request merged**